vibe_code_apps_20/wattlab/energyplus/mcp_runtime.py

```
@@ -0,0 +1,430 @@
+    dest = Path(dest) if dest is not None else default_vendor_target()
+    dest.parent.mkdir(parents=True, exist_ok=True)
+    git = _git_bin()
+    commit = (pin.get("commit") or "").strip()
+    repo = pin["repo"]
+    if not (dest / ".git").is_dir():
+        if dest.exists() and any(dest.iterdir()):
+            raise RuntimeError(
+                f"Vendor path {dest} exists but is not a git clone. "
+                "Remove it or set WATTLAB_ENERGYPLUS_MCP to a clean path."
+            )
+        subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)